CONTRIBUTING.md

```diff
@@ -1,55 +1,69 @@
-This document presents how you can contribute to the OpenRefine project. Please also review our [Governance model](https://github.com/OpenRefine/OpenRefine/blob/master/GOVERNANCE.md)
+The OpenRefine project welcomes contributions in a variety of forms.
+This document contains information a few of the ways you can contribute to the OpenRefine project.
+Please also review our [Governance model](https://github.com/OpenRefine/OpenRefine/blob/master/GOVERNANCE.md)
 
-## Documentation, questions or problems
+## Provide peer user support
 
-Our issue list is only for reporting specific bugs and requesting specific features. If you just don't know how to do something using OpenRefine, or want to discuss some ideas, please
-- try the [user manual](https://openrefine.org/docs/)
-- ask on the [OpenRefine forum](https://forum.openrefine.org/).
+We welcome users to the [OpenRefine forum](https://forum.openrefine.org/) to ask questions and request assistance.
+If you can help answer questions in your area of expertise, it would be a benefit to the community.
 
-
-If you really want to file a bug or request a feature, go to this [issue list](https://github.com/OpenRefine/OpenRefine/issues). Please use the search function first to make sure a similar issue doesn't already exist. 
+If a forum discussion determines there is a bug in OpenRefine or a new feature is identified,
+we welcome bug reports and feature requests. Please search the [issue tracker](https://github.com/OpenRefine/OpenRefine/issues) first to make sure
+the bug / feature hasn't already been added. Note: the development team principally works from the issue
+tracker, so anything not included there risks getting lost.
 
 ## Promote OpenRefine
 
-You don't need to be a coder to contribute to OpenRefine. Did you write a tutorial or article about OpenRefine on your blog or site? Are you organizing a workshop or presentation for OpenRefine in your city? Let us know via our [forum](https://forum.openrefine.org/) or Twitter account ([@OpenRefine](http://twitter.com/OpenRefine)). We will share the news via our monthly update and via our Twitter handle. 
+Promoting OpenRefine is a great way to give back. Did you write a tutorial or article about OpenRefine on your blog or site?
+Are you organizing a workshop or presentation for OpenRefine in your city? Let us know via our [forum](https://forum.openrefine.org/) or Twitter account ([@OpenRefine](http://twitter.com/OpenRefine)).
+We will share the news via our monthly update and via our Twitter handle.
```

Comment:
We are not doing this anymore. See the unresolved conversation regarding our social media presence: https://forum.openrefine.org/t/openrefine-in-the-fediverse/810

Reply:
Done